n2y/main.py

```diff
@@ -33,6 +35,8 @@ def main():
 
     database_id = notion.id_from_share_link(args.database)
 
+    if not args.image_path:
+        args.image_path = args.target
```

Comment:
This works, but it's probably better not to change the `args` object. A better way:

```python
if args.image_path:
    converter.IMAGE_PATH = args.image_path
else:
    converter.IMAGE_PATH = args.target
```

Feel free to change it or keep it as-is. I'm probably getting a little nit-picky.

Reply:
Great point @xaker00. Yeah, @sHermanGriffiths most of the time it's best to avoid mutating a complex object that was generated upstream from your code.